packages/apollo-link-http/README.md

```diff
@@ -53,3 +62,121 @@ client.query({
   }
 })
 ```
+
+### Upgrading from apollo-fetch / apollo-client 
+If you previously used either apollo-fetch or apollo-client, you will need to change the way `use` and `useAfter` are implemented in your app. They can both be implemented in a link like so:
+
+#### Middleware
+
+*Before*
+```js
+// before
+import ApolloClient, { createNetworkInterface } from 'apollo-client';
+
+const networkInterface = createNetworkInterface({ uri: '/graphql' });
+
+networkInterface.use([{
+  applyMiddleware(req, next) {
+    if (!req.options.headers) {
+      req.options.headers = {};  // Create the header object if needed.
+    }
+    req.options.headers['authorization'] = localStorage.getItem('token') ? localStorage.getItem('token') : null;
+    next();
+  }
+}]);
+
+```
+
+*After*
+```js
+import { ApolloLink } from 'apollo-link';
+import { createHttpLink } from 'apollo-link-http';
+
+const httpLink = createHttpLink({ uri: '/graphql' });
+const middlewareLink = new ApolloLink((operation, forward) => {
+  operation.setContext({
+    headers: {
+      authorization: localStorage.getItem('token') || null
+    }
+  });
+  return forward(operation)
+})
+
+// use with apollo-client
+const link = middewareLink.concat(httpLink);
+```
+
+#### Afterware (error)

```

Comment:
I feel like the error & data manipulation examples are duplicative - maybe move the error example to the error link readme instead? It's a great example, just not sure we need two of them